ZUT-PokerPlanningClient/src/app/project-planning/project-planning.module.ts

```diff
@@ -9,10 +9,11 @@ import { SettingsComponent } from './settings/settings.component';
 import { NavMenuComponent } from './nav-menu/nav-menu.component';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { GameItemComponent } from './games/game-item/game-item.component';
+import { GameDeckComponent } from './games/game-deck/game-deck.component';
 
 
 @NgModule({
-  declarations: [ProjectPlanningComponent, GamesComponent, MembersComponent, SettingsComponent, NavMenuComponent, GameItemComponent],
+  declarations: [ProjectPlanningComponent, GamesComponent, MembersComponent, SettingsComponent, NavMenuComponent, GameItemComponent, GameDeckComponent],
```

Comment:
In my opinion we should have a GameModule. 

This way we GameModule could deliver its RoutingModule and
a) Games-Site-Component b) GameItemComponent c) GameDeckComponent...

Creating a GameModule has following advantage: 
-In general we could then easily replace existing GameModule and all of its components in a single place (It would require to replace GameModule to GameModulePremium or something... for example) with Inversion of Control pattern
-GameModulePremium (or something...) we could have different views/functionalities